gui/src/ts/components/data/forceGraph/ForceGraph.tsx

```diff
@@ -82,25 +88,35 @@ export const ForceGraph = <
     );
   }, []);
 
+  React.useEffect(() => {
+    simulation.on("tick", updatePositions).on("end", () => {
+      simulation.on("tick", null);
+    });
+  }, [simulation]);
+
   // Initialize force simulation once and then update node
   // and links with useEffect
-  const simulation = React.useMemo<d3.Simulation<NodeDatum, LinkDatum>>(
```

Comment:
I assume you left this in for a reason rather than nuking it.

Reply:
Kept it for reference but can remove now, removed